pkg/cri/sbserver/podsandbox/controller.go

```diff
@@ -22,6 +22,7 @@ import (
 	"time"
 
 	"github.com/containerd/containerd"
+	eventtypes "github.com/containerd/containerd/api/events"
```

Comment:
What is the purpose of this explicit package name? Keeping the default name of `events` would make code using it more recognizable to devs who have used containerd/api/events before.

Reply:
I think it's for consistency with other usages in the project, but the root cause was that I used `ctrl+c/ctrl+v` 😂 .